packages/docs/src/pages/en/features/display-and-platform.md

```diff
@@ -0,0 +1,457 @@
+
+* **Type:** `boolean`
+* **Default:** `false`
+
+### lgAndDown
+
+This value indicates that the device width less than the **lg** threshold value.
+
+* **Type:** `boolean`
+* **Default:** `false`
+
+### lgAndUp
+
+This value indicates that the device width greater than the **lg** threshold value.
+
+* **Type:** `boolean`
+* **Default:** `false`
+
+### name
+
+The name key correlates to the currently active breakpoint; e.g. **xs**, **sm**, **md**, **lg**, **xl**.
+
+* **Type:** `string`
+* **Default:** `""`
+
+### height
+
+This value is the current window height.
+
+* **Type:** `number`
+* **Default:** `0`
+
+### width
+
+This value is the current window width.
+
+* **Type:** `number`
+* **Default:** `0`
+
+### mobile
+
+This value indicates whether the current display width is less than the defined [mobileBreakpoint](#mobile-breakpoint)
+
+* **Type:** `boolean`
+* **Default:** `false`
+
+### mobileBreakpoint
+
+This value indicates whether the current display width is less than the defined [mobileBreakpoint](#mobile-breakpoint)
+
+* **Type:** `number | string`
+* **Default:** `"md"`
+
+### platform
+
+This object contains information from the detected device's [userAgent](https://developer.mozilla.org/en-US/docs/Web/API/NavigatorID/userAgent) and whether it supports [touch]() and [intersection]().
+
+* **Type:** `object`
+* **Default:**
+
+  ```js
+    {
+      android: false,
+      ios: false,
+      cordova: false,
+      electron: false,
+      edge: false,
+      firefox: false,
+      opera: false,
+      win: false,
+      mac: false,
+      linux: false,
+      intersection: false,
+      touch: false,
+      ssr: false,
+    }
+  ```
+
+### scrollbarWidth
+
+This value is the width of the browser scrollbar.
+
+* **Type:** `number`
+* **Default:** `16`
+
+### thresholds
+
+This object is used for device viewport calculations.
+
+* **Type:** `object`
+* **Default:**
+
+  ```js
+    {
+      xs: 600,
+      sm: 960,
+      md: 1280,
+      lg: 1920,
+      xl: 3840,
+    }
+  ```
+
+* **Usage:**
+
+  ```js
+    // Composables
+    import { useDisplay } from 'vuetify/lib/composables/display'
+
+    export default {
+      mounted () {
+        const display = useDisplay()
+
+        // display thresholds are not reactive
+        // and do not need to use .value
+        console.log(display.thresholds.md) // 1280
+      }
+    }
+  ```
+
+## Using Setup
+
+Setup is a new feature of Vue 3 that enables more control over the
+
+```html
+<template>
+  <v-dialog :fullscreen="display.mobile.value">
+    ...
+  </v-dialog>
+</template>
+
+<script>
+  // Composables
+  import { useDisplay } from 'vuetify/lib/compsables/display'
+
+  export default {
+    setup () {
+      const display = useDisplay()
+
+      return { display }
+    },
+  }
+</script>
+```
+
+### JSX
+
+Using JSX with setup can accomplish the same thing as a template. The following example shows how to bind the same **fullscreen** property, but using JSX:
```

Comment:
```suggestion
Using JSX with the `setup` function you can accomplish the same thing as a template. The following example shows how to bind the **display** composable to the same **fullscreen** property of `v-dialog`, but using JSX:
```

Reply:
Not sure if this section should even be here? Why would display composable specifically get example showing JSX syntax, when it's not used anywhere else in docs?

Reply:
I added it because the intention was to add it in other areas as a separate PR. However, it probably should be done as a separate PR for everything. Will remove this section.